src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -344,6 +344,22 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
                 return Promise.resolve();
             }));
 
+        if (await this.driver.isGeneratedColumnsSupported(this.mode)) {
+            // if table have column with generated type, we must add the expression to the meta table
+            await Promise.all(table.columns
+                .filter(column => column.generatedType === "STORED" && column.asExpression)
+                .map(async column => {
+                    const tableName = await this.getTableNameWithSchema(table.name);
```

Comment:
Can't this be done outside the `map` call - then all of this can be sync - no need for all of the `Promise.all` and async.

Reply:
I mean I could transform it to a for loop. But does this make it better? I mean we need to do at least one async call. Is it really important where?